ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml

```diff
@@ -0,0 +1,89 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.17"
+  tools:
+    name: "4.15"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    release:
+      architecture: multi
+      channel: candidate
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 1Gi
+tests:
+- as: azure-ipi-oidc-mixarch-f7-logging
```

Comment:
4.15 release every 3 weeks. Perhaps we can use lower test frequency in 4.15 ?

Reply:
@liangxia  It is Openshift Logging 6.1 test, not for 4.15. Logging will be release at Nov 13. we can use  lower test frequency  after that.

Reply:
The other logging version 5.8/5.9/6.0 with OCP 4.15 are all in `f7`
```
openshift-openshift-tests-private-release-4.15__amd64-nightly-openshift-logging-5.9.yaml:- as: azure-ipi-proxy-workers-rhcos-rhel8-f7-logging
openshift-openshift-tests-private-release-4.15__amd64-nightly-openshift-logging-5.9.yaml:- as: aws-ipi-f7-disruptive-logging
openshift-openshift-tests-private-release-4.15__multi-nightly-openshift-logging-5.8.yaml:- as: aws-ipi-ovn-ipsec-arm-mixarch-f7-logging
openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.0.yaml:- as: azure-ipi-oidc-mixarch-f7-logging
```

Reply:
Logging z-stream will be released every sprint (3 week).   Currently, we support 5.8,5.9,6.0. 6.1 is the next major release.

Reply:
There are about 4*4 =16 jobs for Logging ever week.

Reply:
For logging 6.1, I'm fine with testing it weekly (f7).  
I don't understand for the others.  E.g, ocp 4.15+logging 5.9, both ocp 4.15 and logging 5.9 release every 3 weeks, why it's required to test every weeks ?
CC @jianlinliu as this is cloud cost related.

Reply:
@liangxia  Can you merge this？ I can decrease the frequency to bi-week for Logging 5.8,5.9 and 6.1 in another PR.

Reply:
Ack